server/routes/api/settings.ts

```diff
@@ -74,6 +75,42 @@ settingsRoutes.route("/qr_enabled")
 		}
 	});
 
+settingsRoutes.route("/admin_emails")
+	.put(isAdmin, uploadHandler.any(), async (request, response) => {
+		let rawAdminString = request.body.adminString;
+		let addAdmins = request.body.addAdmins === "true";
+
+		if (!rawAdminString) {
+			return response.status(400).json({
+				"error": "Invalid value for admin emails"
+			});
+		}
+
+		let adminEmailArray = rawAdminString.split(",").map((element: string) => {
```

Comment:
Can you make this a regex of `/, ?/` so that it will split on \<comma\>\<space\> in addition to just \<comma\>?

Reply:
@petschekr just updated, used `/, */` instead so if there are multiple spaces between emails, it gets fixed